DO NOT UPSTREAM

This patch breaks the build of multiple (all?) targets

The link error is cpuidle_devices is undefined.

Not-Signed-Off-By: Mark Charlebois <charlebm@gmail.com>

--- linux.orig/Makefile
+++ linux/Makefile
@@ -700,7 +700,10 @@
 # source of a reference will be _MergedGlobals and not on of the whitelisted names.
 # See modpost pattern 2
 KBUILD_CFLAGS += $(call cc-option, -mno-global-merge,)
-KBUILD_CFLAGS += $(call cc-option, -fcatch-undefined-behavior)
+#KBUILD_CFLAGS += $(call cc-option, -fcatch-undefined-behavior)
+KBUILD_CFLAGS += $(call cc-option, -fsanitize=undefined-trap)
+KBUILD_CFLAGS += $(call cc-option, -fsanitize-undefined-trap-on-error)
+
 else
 
 # This warning generated too much noise in a regular build.
